samples/c/common/opencv_c_wrapper/bmp_reader.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+
+typedef struct BmpHeaderType {
+    unsigned short type;             /* Magic identifier            */
+    unsigned int size;               /* File size in bytes          */
+    unsigned int reserved;
+    unsigned int offset;             /* Offset to image data, bytes */
+} BmpHeader;
+
+
+typedef struct BmpInfoHeaderType {
+    unsigned int size;               /* Header size in bytes      */
+    int width, height;               /* Width and height of image */
+    unsigned short planes;           /* Number of colour planes   */
+    unsigned short bits;             /* Bits per pixel            */
+    unsigned int compression;        /* Compression type          */
+    unsigned int imagesize;          /* Image size in bytes       */
+    int xresolution, yresolution;    /* Pixels per meter          */
+    unsigned int ncolours;           /* Number of colours         */
+    unsigned int importantcolours;   /* Important colours         */
+} BmpInfoHeader;
+
+
+typedef struct BitMapType {
+    BmpHeader header;
+    BmpInfoHeader infoHeader;
+    int width, height;
+    unsigned char* data;
+} BitMap;
+
+
+int readBmpImage(const char* fileName, BitMap* image);
```

Comment:
is it possible not to copy code, but `format_reader_c` will just wrap `format_reader`?

Reply:
@ilya-lavrenov do you mean creating common folder under sample subfolder to be shared between c/cpp samples ?

Reply:
or we can install this common part to both C and C++ places like we do for CMakeLists.txt https://github.com/openvinotoolkit/openvino/blob/master/samples/CMakeLists.txt#L61-L63